packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -25,10 +27,17 @@ const TradersHubOnboarding = observer(() => {
                         data_testid='dt_traders_hub_onboarding_icon'
                         icon={is_dark_mode_on ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
                         size={20}
-                        onClick={() => {
-                            history.push(routes.onboarding);
-                            setIsOnboardingVisited(false);
-                        }}
+                        onClick={
+                            is_next_wallet_enabled
+                                ? () => {
+                                      localStorage.setItem('walletsOnboarding', 'started');
```

Comment:
whats the purpose of using localStorage here? is it just to keep the state while onboarding, or is it here to indicate that someone already had onboarding?

Reply:
@sergei-deriv it's only to indicate onboarding started and trigger `react-joyride` to show onboarding steps

Reply:
@wojciech-deriv it's the card related to this feature https://app.clickup.com/t/20696747/WALL-2239